docs/diagrams/ParserPartialInitialisationSequenceDiagram.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":ProjectCommandParser" as ProjectCommandParser LOGIC_COLOR
+participant ":AddProjectCommand" as AddProjectCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":ProjectWithoutModel" as ProjectWithoutModel MODEL_COLOR
+participant ":Project" as Project MODEL_COLOR
+end box
+
+--> ProjectCommandParser
+activate ProjectCommandParser
+
+create ProjectWithoutModel
+ProjectCommandParser --> ProjectWithoutModel : new ProjectWithoutModel(name, ...)
+activate ProjectWithoutModel
+
+ProjectWithoutModel --> ProjectCommandParser
+deactivate ProjectWithoutModel
+
+create AddProjectCommand
+ProjectCommandParser --> AddProjectCommand : AddProjectCommand(projectWithoutModel)
+activate AddProjectCommand
+
+AddProjectCommand --> ProjectCommandParser
+deactivate AddProjectCommand
+
+activate LogicManager
```

Comment:
Activation bar shouldnt begin before execute is executed based on the image

Reply:
Addressed